java-frontend/src/main/java/org/sonar/java/JavaClasspath.java

```diff
@@ -20,20 +20,20 @@
 package org.sonar.java;
 
 import com.google.common.collect.Lists;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.sonar.api.batch.fs.FileSystem;
 import org.sonar.api.batch.fs.InputFile;
 import org.sonar.api.config.Settings;
 import org.sonar.api.resources.Project;
-import org.sonar.api.utils.TimeProfiler;
+import org.sonar.api.utils.log.Logger;
+import org.sonar.api.utils.log.Loggers;
+import org.sonar.api.utils.log.Profiler;
 
 import java.io.File;
 import java.util.List;
 
 public class JavaClasspath extends AbstractJavaClasspath {
 
-  private static final Logger LOG = LoggerFactory.getLogger(JavaClasspath.class);
+  private static final Logger LOG = Loggers.get(JavaClasspath.class);
 
   public JavaClasspath(Project project, Settings settings, FileSystem fs) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "Project"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)